Modelica/Units.mo

```diff
@@ -342,6 +342,10 @@ end UsersGuide;
         displayUnit="bar");
     type AbsolutePressure = Pressure (min=0.0, nominal = 1e5);
     type PressureDifference = Pressure;
+    type PressureRate = Real (
```

Comment:
At some point I think we need to review the logic of units, as we are adding more and more, and the question is whether we need a new type here, or we could just declare the variable with Pa/s.

Reply:
I am fine with either, since these are my first contribution to the MSL, I was just trying to match the current style.

Reply:
Since quantities must match in connections, the habit of defining types in the MSL where the `quantity` is given is a good measure for avoiding that the same conceptual quantity is expressed with different `quantity` strings in different parts of a library, or in different libraries.